src/utils.js

```diff
@@ -187,10 +187,10 @@ var Utils = exports.Utils = {
    * @returns {Boolean} true if disabled
    **/
   isPrivacyBadgerEnabled: function(origin){
-    if(localStorage.disabledSites && JSON.parse(localStorage.disabledSites).length > 0){
+    if(localStorage.disabledSites){
       var sites = JSON.parse(localStorage.disabledSites);
-      for(var i = 0; i < sites.length; i++){
-        if(sites[i] === origin){ return false; }
+      if(sites[origin]){
+        return false;
```

Comment:
We could happily write to localStorage each time we update the object in memory as long as we do it asynchronously.